openqa-investigate

```diff
@@ -151,6 +161,29 @@ investigate() {
         echoerr "Job already has a clone, skipping investigation. Use the env variable 'force=true' to trigger investigation jobs"
         return 0
     fi
+
+    # skip parallel children (will be handled together with parallel parent)
+    # note: For this to work, this script needs to be executed for all job results (so we won't miss the investigation in case the parallel parent has passed).
+    parallel_parents=$(echo "$job_data" | runjq -r '(.job.parents["Parallel"] | length)') || return $?
+    [[ $parallel_parents != 0 ]] \
+        && echoerr "Skipping investigation of job $id: it will be handled together with its parallel parent" && return 0
+
+    # postpone if not all dependencies are done/cancelled
+    dependency_data=$(openqa-cli "${client_args[@]}" --apibase '' --json tests/"$id"/dependencies_ajax)
```

Comment:
if we need this hack how about extending the openQA API as needed?

Reply:
I could move the route into the API namespace. For now I would refrain from it (since everything is still a bit experimental and we don't need to commit to a certain API for "real external" users yet). Of course we need to take care that we don't break this route without also adjusting this script but I suppose that this mistake is quite unlikely to happen.

Reply:
Ok, leave it as is for now